specification/containerinstance/resource-manager/readme.md

```diff
@@ -151,85 +151,6 @@ python:
   output-folder: $(python-sdks-folder)/azure-mgmt-containerinstance
 ```
 
-
-## Go
-
-These settings apply only when `--go` is specified on the command line.
-
-``` yaml $(go)
-go:
-  license-header: MICROSOFT_APACHE_NO_VERSION
-  namespace: containerinstance
-  clear-output-folder: true
-```
-
-### Go multi-api
-
-``` yaml $(go) && $(multiapi)
-batch:
-  - tag: package-2018-06
-  - tag: package-2018-04
-  - tag: package-2018-02-preview
-  - tag: package-2017-12-preview
-  - tag: package-2017-10-preview
-  - tag: package-2017-08-preview
-```
-
-### Tag: package-2018-06 and go
-
-These settings apply only when `--tag=package-2018-06 --go` is specified on the command line.
-Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
-
-``` yaml $(tag) == 'package-2018-04' && $(go)
```

Comment:
Note that this uses the wrong tag, this is what was fixed in the new config file.